getting_started/workflow/assets/importing_scenes.rst

```diff
@@ -52,6 +53,24 @@ Godot provides a `Python
 Plugin <https://github.com/godotengine/collada-exporter>`__
 that will do a much better job of exporting the scenes.
 
+Exporting GLTF 2.0 files from Blender
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are 3 ways to export GLTF files from Blender. As a GLTF binary (.glib file), GLTF embedded (.gltf file), 
```

Comment:
It's `.glb`, not `.glib` IIRC.

Also it would be nice to use inline code formatting for extensions:
<code>
``.gltf``
</code>

And to fix the capitalization of glTF, the "gl" part is lowercase (the issue existed already in this tutorial though).